.mypy/proper_plugin.json

```diff
@@ -0,0 +1 @@
ult_enabled": true}, "blocker": false, "only_once": false, "allow_dups": false, "origin": ["/mypy/errors.py", 762, 762], "target": "mypy.errors.Errors.sort_messages"}, {".class": "mypy.errors.ErrorInfo", "import_ctx": [["misc/proper_plugin.py", 1], ["/mnt/d/Users/Morgan/projects/mypy/mypy/plugin.py", 133], ["/mnt/d/Users/Morgan/projects/mypy/mypy/messages.py", 22]], "file": "/mypy/errors.py", "module": "mypy.errors", "type": "Errors", "function_or_member": "sort_messages", "line": 762, "column": 60, "severity": "error", "message": "Expression has type \"Any\"", "code": {".class": "mypy.errorcodes.ErrorCode", "code": "misc", "description": "Miscellaneous other checks", "category": "General", "default_enabled": true}, "blocker": false, "only_once": false, "allow_dups": false, "origin": ["/mypy/errors.py", 762, 762], "target": "mypy.errors.Errors.sort_messages"}, {".class": "mypy.errors.ErrorInfo", "import_ctx": [["misc/proper_plugin.py", 1], ["/mnt/d/Users/Morgan/projects/mypy/mypy/plugin.py", 133], ["/mnt/d/Users/Morgan/projects/mypy/mypy/messages.py", 22]], "file": "/mypy/errors.py", "module": "mypy.errors", "type": "Errors", "function_or_member": "sort_messages", "line": 762, "column": 51, "severity": "error", "message": "Expression type contains \"Any\" (has type \"Tuple[Any, Any]\")", "code": {".class": "mypy.errorcodes.ErrorCode", "code": "misc", "description": "Miscellaneous other checks", "category": "General", "default_enabled": true}, "blocker": false, "only_once": false, "allow_dups": false, "origin": ["/mypy/errors.py", 762, 762], "target": "mypy.errors.Errors.sort_messages"}, {".class": "mypy.errors.ErrorInfo", "import_ctx": [["misc/proper_plugin.py", 1], ["/mnt/d/Users/Morgan/projects/mypy/mypy/plugin.py", 133], ["/mnt/d/Users/Morgan/projects/mypy/mypy/messages.py", 22]], "file": "/mypy/errors.py", "module": "mypy.errors", "type": "Errors", "function_or_member": "sort_messages", "line": 762, "column": 41, "severity": "error", "message": "Expression type contains \"Any\" (has type \"Callable[[Any], Tuple[Any, Any]]\")", "code": {".class": "mypy.errorcodes.ErrorCode", "code": "misc", "description": "Miscellaneous other checks", "category": "General", "default_enabled": true}, "blocker": false, "only_once": false, "allow_dups": false, "origin": ["/mypy/errors.py", 762, 762], "target": "mypy.errors.Errors.sort_messages"}, {".class": "mypy.errors.ErrorInfo", "import_ctx": [["misc/proper_plugin.py", 1], ["/mnt/d/Users/Morgan/projects/mypy/mypy/plugin.py", 133], ["/mnt/d/Users/Morgan/projects/mypy/mypy/messages.py", 22]], "file": "/mypy/errors.py", "module": "mypy.errors", "type": "Errors", "function_or_member": "save_baseline", "line": 807, "column": 34, "severity": "error", "message": "Expression has type \"Any\"", "code": {".class": "mypy.errorcodes.ErrorCode", "code": "misc", "description": "Miscellaneous other checks", "category": "General", "default_enabled": true}, "blocker": false, "only_once": false, "allow_dups": false, "origin": [
```

Comment:
full form paths

Reply:
where?

Reply:
![image](https://user-images.githubusercontent.com/57028336/144543179-3b2f30a4-2452-4719-9d9e-688d4b12e2d9.png)

Reply:
Added that one to the filter function